keras/activations.py

```diff
@@ -15,8 +15,8 @@ def softmax(x, axis=-1):
     """Softmax activation function.
 
     # Arguments
-        x : Tensor.
-        axis: Integer, axis along which the softmax normalization is applied.
+        - x : Input tensor.
+        - axis: Integer, axis along which the softmax normalization is applied.
```

Comment:
Don't structure arguments as a bullet-point list, each line should start with the argument name.

Reply:
Not doing this results in formatting that looks bad. See for example the current documentation of the softmax function in https://keras.io/activations/ .